### ISSUE TITLE
[Feature Request] New Page for Status

### ISSUE THREAD

**Issue by kuckuckmann** [joBr99/nspanel-lovelace-ui#532]
### FEATURE DESCRIPTION
Eine neue Seite die entweder am unteren Rand einen Bereich für ein kleineres Auswahl-Icon hat, so ähnlich wie es auf der ThermoCard ist. Diese kleineren Icons sollen aus Auswahl dienen.
Im Haupten gibt es dann eine gleichmäßige Anzahl an Kacheln (ich würde 4 Vorschlagen), in denen dann verschiedene Werte angezeigt werden können.

Als Beispiel hätte ich Temperatursensoren. Im unteren Menüband wird ein Thermometer als Icon dargestellt, wird dieses angeklickt wird in den 4 Kacheln die Daten von bis zu 4 Sensoren angezeigt. Um auch z.B. Tür/Fenstersensoren abbilden zu können, muss es die Möglichkeit geben in den Kacheln so wohl Text als auch ein Icon (für z.B. Fenster / Tür-Darstellung) zu zeigen.

Die Kacheln bestehen aus drei Teilen. Oben ein Label, dann die eigentliche Kachel in der ei Textwert oder Icon dargestellt werden kann und unten ein band in dem z.B. Batterieanzeige oder Signalstärke wieder geben kann. Ggf. kann man dafür auch statt einem zwei Boxen dort platzieren.

In der Hauptkachel den Textwert Rollieren zu lassen, so das abwechselnd verschiedene Werte wie Temperatur, Luftfeuchtigkeit, etc. dargestellt werden können, ist vermutlich eine Abhängigkeit des Skriptes dahinter.

### ADDITIONAL CONTEXT
Beispiels maske:

![image](https://user-images.githubusercontent.com/99131208/197397475-ed4142c0-3b0c-48a5-be29-6e1e1aab1c2b.png)

### PANEL / FIRMWARE VERION
EU-Panel, DE, Tasmota 12.1.0, v3.4.0.4

**Comment by joBr99** [comment]
Hört sich eher nach cardGrid mit zusätzlichen Buttons unten an. 

Kein Plan wie sich das unter ioBroker verhält, mit HA werden sensoren aktuell so dargestellt:

![image](https://user-images.githubusercontent.com/29555657/197398868-d922200a-a176-4aa3-a755-cf4e14da624f.png)

![image](https://user-images.githubusercontent.com/29555657/197398894-8c8d24b9-cbfa-43b7-9c20-4c82e468e2d1.png)

**Comment by kuckuckmann** [comment]
Ja, natürlich kann man das bedingt mit einer cardGrid abbilden, aber dann benötige ich pro Sensor eine Karte um die Werte Temperatur, Luftfeuchtigkeit, Batteriestatus und Signalstärke abzubilden.  
Mit dieser Idee kann ich dann direkt mehrere Sensoren abbilden, hab einen Vergleich auf einen Blick. Denken wir mal an den Fall Tür-/Fenstersensoren - Du willst die Alarmanalage scharf machen aber es ist noch ein Sensor auf offen. Entweder klicke ich mich durch diverse Karten um zu suchen welchen Sensor es betrifft oder ich kann es auf eine Karte reduzieren, weil ich alle auf ein und derselben Karte sehen kann. Stichwort Komfort ;-)

**Comment by joBr99** [comment]
Für den Fall Alarmanlage/Fensterkontakte sollte es ne Option geben, um nur offene Kontakte anzuzeigen, das ist noch komfortabler :)

Hier, wird das entity z.B. nur angezeigt, wenn der State "on" ist, geschlossene Fensterkontakte sind mir in dem Fall ja egal, der Status von der Gruppe lässt sich grundsätzlich auch auf cardAlarm darstellen, inkl navigation von der cardAlarm zu dieser Seite.
```
      - type: cardGrid
        title: Fensterkontakte
        entities:
          - entity: binary_sensor.sensor_bad_contact
            state: "on"
          - entity: binary_sensor.sensor_fenster_links_contact
            state: "on"
```

**Comment by kuckuckmann** [comment]
Ja, das mag sein dass dies im Falle der Alarmanlage eine komfortable Möglichkeit ist. Es war aber nur ein Beispiel.
Letztlich geht es mir um die Möglichkeit die Werte mehrerer Sensoren auf einer card dazustellen.

**Comment by kuckuckmann** [comment]
Hm, es ist hier sehr ruhig. Ihr seid bestimmt schon fleißig an der Umsetzung meines Wunsches dran :-)

**Comment by Armilar** [comment]
> Hört sich eher nach cardGrid mit zusätzlichen Buttons unten an.
> 
> Kein Plan wie sich das unter ioBroker verhält, mit HA werden sensoren aktuell so dargestellt:
> 
> ![image](https://user-images.githubusercontent.com/29555657/197398868-d922200a-a176-4aa3-a755-cf4e14da624f.png)
> 
> ![image](https://user-images.githubusercontent.com/29555657/197398894-8c8d24b9-cbfa-43b7-9c20-4c82e468e2d1.png)

Die cardGrid ist im ioBroker auch nicht anders
![image](https://user-images.githubusercontent.com/102996011/216039877-e676952a-ac98-4ea9-9c89-29ac89fe8a9e.png)

sogar etwas weiter, denn es lassen sich bis zu 12 Zustände visualisieren:
![image](https://user-images.githubusercontent.com/102996011/216040059-36026d7f-e1c5-460b-9e8f-d2ffac09e1e0.png)

Wenn du aber gezielt, und da gab es bereits einige Anfragen, diverse Zustände eines virtuellen Devices visualisieren möchtest, geht das zwar auch, jedoch sieht es eher kompliziert aus...

Nehmen wir mal das Beispiel einer herkömmlichen Steckdose mit diversen Parametern:
![image](https://user-images.githubusercontent.com/102996011/216041667-cdbcf52d-8f6a-4bf8-9888-a6b3998b1588.png)

Die könnten faktisch in einem Zuge aus "einem" Alias importiert und visualisiert werden. Gestaltet sich allerdings in einer cardGrid sehr schwierig.

Andererseits, wenn es ein Info-Popup wäre, den man auch aus diversen Entitäten aus öffnen könnte (z.B. aus dem popupLight), um dann zusätzliche Infos zum jeweiligen Device anzeigen zu können, wäre das aus meiner Sicht ein Vorteil. Also aus definierten Popup's ein weiteres Info-Popup für diverse Info-Zustände öffnen.

Ebenfalls aus einer CardGrid heraus, um die restlichen Zustände eines Sensors (z.B. Batteriezustände oder Erreichbarkeit) bei z.B. Zigbee-Devices bei Bedarf mit einblenden zu können (kann man natürlich auch durch gezieltes aufrufen einer Subpage erreichen)
